<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_NagViewItem</class>
 <widget class="QWidget" name="C_NagViewItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>C_NagViewItem</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="stw_opensyde_gui_elements::C_OgeGbxNavigationTab" name="pc_GroupBoxStyle">
     <property name="title">
      <string>StyleBox</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="stw_opensyde_gui_elements::C_OgeGbxNavigationSectionSelected" name="pc_GroupBoxActive">
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="stw_opensyde_gui_elements::C_OgePubNavigationHover" name="pc_WidgetTopButton" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>43</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>43</height>
              </size>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0,1,1,0">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>8</number>
              </property>
              <property name="topMargin">
               <number>9</number>
              </property>
              <property name="rightMargin">
               <number>7</number>
              </property>
              <property name="bottomMargin">
               <number>9</number>
              </property>
              <item>
               <widget class="stw_opensyde_gui_elements::C_OgeLabSvgOnly" name="pc_LabelExpand">
                <property name="minimumSize">
                 <size>
                  <width>16</width>
                  <height>16</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16</width>
                  <height>16</height>
                 </size>
                </property>
                <property name="text">
                 <string>Expand</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>3</width>
                  <height>0</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="stw_opensyde_gui_elements::C_OgeLabSvgOnly" name="pc_LabelError">
                <property name="minimumSize">
                 <size>
                  <width>16</width>
                  <height>16</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16</width>
                  <height>16</height>
                 </size>
                </property>
                <property name="text">
                 <string>Error</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>6</width>
                  <height>0</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="stw_opensyde_gui_elements::C_OgeLabGenericNoPaddingNoMargins" name="pc_LabelHeading">
                <property name="text">
                 <string>Label</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="stw_opensyde_gui_elements::C_OgeLeNavigation" name="pc_LineEditHeading"/>
              </item>
              <item>
               <widget class="stw_opensyde_gui_elements::C_OgeGbxTransparent" name="pc_GroupBoxHoverButtons">
                <property name="title">
                 <string/>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <property name="spacing">
                  <number>10</number>
                 </property>
                 <property name="leftMargin">
                  <number>7</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="stw_opensyde_gui_elements::C_OgePubSvgIconOnly" name="pc_PushButtonEdit">
                   <property name="minimumSize">
                    <size>
                     <width>16</width>
                     <height>16</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16</width>
                     <height>16</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Edit</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="stw_opensyde_gui_elements::C_OgePubSvgIconOnly" name="pc_PushButtonCopy">
                   <property name="minimumSize">
                    <size>
                     <width>16</width>
                     <height>16</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16</width>
                     <height>16</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Copy</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="stw_opensyde_gui_elements::C_OgePubSvgIconOnly" name="pc_PushButtonDelete">
                   <property name="minimumSize">
                    <size>
                     <width>16</width>
                     <height>16</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16</width>
                     <height>16</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Delete</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="stw_opensyde_gui_elements::C_OgePubNavigationHover" name="pc_WidgetSetup" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="stw_opensyde_gui_elements::C_OgePubNavigationHover" name="pc_WidgetUpdate" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="stw_opensyde_gui_elements::C_OgePubNavigationHover" name="pc_WidgetDashboard" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>76</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeGbxTransparent</class>
   <extends>QGroupBox</extends>
   <header>C_OgeGbxTransparent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeGbxNavigationTab</class>
   <extends>QGroupBox</extends>
   <header>C_OgeGbxNavigationTab.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeLabGenericNoPaddingNoMargins</class>
   <extends>QLabel</extends>
   <header>C_OgeLabGenericNoPaddingNoMargins.h</header>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeLabSvgOnly</class>
   <extends>QLabel</extends>
   <header>C_OgeLabSvgOnly.h</header>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgePubSvgIconOnly</class>
   <extends>QPushButton</extends>
   <header>C_OgePubSvgIconOnly.h</header>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgePubNavigationHover</class>
   <extends>QWidget</extends>
   <header>C_OgePubNavigationHover.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeLeNavigation</class>
   <extends>QLineEdit</extends>
   <header>C_OgeLeNavigation.h</header>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeGbxNavigationSectionSelected</class>
   <extends>QGroupBox</extends>
   <header>C_OgeGbxNavigationSectionSelected.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
